### ISSUE TITLE
Anchor contrast

### ISSUE THREAD

**Issue by marcsoler** [denysvitali/thebestmotherfuckingwebsite#50]
The website fails the [WCAG conformance](https://www.w3.org/WAI/WCAG2AAA-Conformance.html) level AAA. The anchor tags require a higher contrast even on the high-contrast mode.

Suggestion: #03F instead of #0051c9.

Website is Accessible. But not that much ;)

**Comment by denysvitali** [comment]
Fixed in an upcoming commit (:
